src/lib/ds.min.js

```diff
@@ -1 +1,3 @@
+/* eslint-disable */
```

Comment:
Shouldn't this be `eslint-enable`?

Reply:
Actually, it might be better to create a .eslintignore file and list this file there. This way we don't have to remember to add the comments if we update ds.min.js

Reply:
> Shouldn't this be `eslint-enable`?

Just removed that since we aim to disable eslint for the whole file here.

> Actually, it might be better to create a .eslintignore file and list this file there. This way we don't have to remember to add the comments if we update ds.min.js

Yeah, actually that was the first solution that came to me, and then I find that this only works for `npm run lint`, somehow  it ignores `.eslintignore` when `npm run build`.
![image](https://user-images.githubusercontent.com/43995067/182267551-8f518cc8-0489-4451-9319-f6a79ab89c70.png)
![](https://user-images.githubusercontent.com/43995067/182267951-bac6fd84-c612-46a1-b87c-9b9d24a16176.png)